trueline.sh

```diff
@@ -593,6 +596,11 @@ if [[ -z "$TRUELINE_WORKING_DIR_ABBREVIATE_PARENT_DIRS_LENGTH" ]]; then
     TRUELINE_WORKING_DIR_ABBREVIATE_PARENT_DIRS_LENGTH=1
 fi
 
+# Prompt
```

Comment:
Move this before the vimode config entry and replace the `# Prompt` comment with `# Trailing separator`